core/src/main/java/org/elasticsearch/index/query/TermsQueryBuilder.java

```diff
@@ -126,6 +130,26 @@ public TermsQueryBuilder execution(String execution) {
     }
 
     /**
+     * Sets the minimum number of matches across the provided terms. Defaults to <tt>1</tt>.
+     * @deprecated use [bool] query instead
+     */
+    @Deprecated
+    public TermsQueryBuilder minimumShouldMatch(String minimumShouldMatch) {
+        this.minimumShouldMatch = minimumShouldMatch;
+        return this;
+    }
+
+    /**
+     * Disables <tt>Similarity#coord(int,int)</tt> in scoring. Defaults to <tt>false</tt>.
+     * @deprecated use [bool] query instead
+     */
+    @Deprecated
+    public TermsQueryBuilder disableCoord(boolean disableCoord) {
```

Comment:
Thanks for doing this, can you also make the parser use a ParseField that marks it as deprecated?